pages/login.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="login-page">
+    <v-btn
+      rounded
+      fab
+      small
+      icon
+      :ripple="false"
+      class="nav-back-btn"
+      @click="to"
+    >
+      <v-icon>mdi-arrow-left-circle-outline</v-icon>
+    </v-btn>
+    <div class="login-form">
+      <v-tabs v-model="currentTab">
+        <v-tab :ripple="false" v-for="tab in tabs" :key="tab">{{ tab }}</v-tab>
+      </v-tabs>
+
+      <v-tabs-items v-model="currentTab">
+        <v-tab-item>
+          <form class="login-form" @submit.prevent="submitHandler">
+            <div class="login-form-content">
+              <input
+                type="email"
+                required
+                placeholder="Email"
+                class="login-row py-3 px-5 w-full bg-white rounded-lg"
+                v-model="form.email"
+              />
+              <input
+                type="password"
+                required
+                placeholder="Password"
+                class="login-row py-3 px-5 w-full bg-white rounded-lg"
+                v-model="form.password"
+              />
+              <a href="#" class="login-row forgot-password-btn pl-[18px] link"
+                >Forgot password?</a
+              >
+              <div class="login-row flex flex-col">
+                <a
+                  href="#"
+                  class="
+                    external-auth-btn
+                    py-3
+                    px-5
+                    w-full
+                    bg-white
+                    rounded-lg
+                    mb-[10px]
+                    text-center
+                  "
+                  >Sign in with Google</a
+                >
+                <a
+                  href="#"
+                  class="
+                    external-auth-btn
+                    py-3
+                    px-5
+                    w-full
+                    bg-white
+                    rounded-lg
+                    text-center
+                  "
+                  >Sign in with Facebook</a
+                >
+              </div>
+              <p class="login-row text-center mb-0">
+                New to Map of Interaction?
+                <a href="#" class="join-btn link">Join now</a>
```

Comment:
I think it should be an internal link . Use <NuxtLink to="/">Join now</NuxtLink> for that

Reply:
i will leave like this for now